ibctest/relayer.go

```diff
@@ -0,0 +1,256 @@
+package ibctest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/relayer/v2/cmd"
+	"github.com/cosmos/relayer/v2/internal/relayertest"
+	"github.com/cosmos/relayer/v2/relayer/provider/cosmos"
+	"github.com/ory/dockertest/v3"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+)
+
+type Relayer struct {
```

Comment:
This is good, I wanted to move this struct to its own file but never got around to it.

I assume everything here was a direct copy and paste.

Reply:
yes, all a copy paste except the new interface methods `FlushAcknowledgements`, `FlushPackets`, and `GetWallet` and `opts` added to `LinkPath`